packages/frontend-2/components/viewer/AnchoredPoints.vue

```diff
@@ -86,30 +86,53 @@
       "
     >
       <div
-        class="w-full h-full outline -outline-offset-0 outline-8 rounded-md outline-blue-500/40"
+        class="w-full h-full outline -outline-offset-0 outline-8 rounded-md outline-blue-500"
       >
-        <div class="absolute bottom-4 right-4 p-2 pointer-events-auto">
-          <Portal to="pocket-right">
-            <FormButton
-              v-if="spotlightUserSessionId && spotlightUser"
-              size="xs"
-              class="truncate"
-              @click="() => (spotlightUserSessionId = null)"
-            >
-              <span>Stop Following {{ spotlightUser?.userName.split(' ')[0] }}</span>
-            </FormButton>
+        <div class="absolute top-0 left-[50%] -translate-x-1/2 flex">
+          <svg
+            class="mt-1"
+            width="8"
+            height="8"
+            viewBox="0 0 8 8"
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <path d="M0 0C4.5 0 8 3.5 8 8V0H0Z" class="fill-blue-500" />
+          </svg>
+          <div
+            class="pointer-events-auto bg-primary text-white text-xs px-3 h-8 flex items-center rounded-b-md cursor-default"
+          >
+            <div v-if="spotlightUserSessionId && spotlightUser">
+              Following {{ spotlightUser?.userName.split(' ')[0] }}
+              <FormButton
+                color="secondary"
+                size="xs"
+                class="ml-1 -mr-1.5"
+                @click="() => (spotlightUserSessionId = null)"
+              >
+                <span>Stop</span>
+              </FormButton>
+            </div>
             <div
               v-else
-              v-tippy="followers.map((u) => u.name).join(', ')"
-              class="text-xs p-2 font-bold text-primary"
+              v-tippy="{ placement: 'bottom' }"
+              :content="followers.map((u) => u.name).join(', ')"
             >
               Followed by {{ followers[0].name.split(' ')[0] }}
               <span v-if="followers.length > 1">
                 & {{ followers.length - 1 }}
                 {{ followers.length - 1 === 1 ? 'other' : 'others' }}
               </span>
             </div>
-          </Portal>
+          </div>
+          <svg
+            class="mt-1"
```

Comment:
```suggestion
            class="absolute right-[0.5px] top-1"
```

Reply:
@andrewwallacespeckle These changes break the layout for me. Why do this instead of the top margin?

Reply:
Sorry, you will need to add `px-2` to the parent to add padding. 

Margin works, but when the item is being absolutely positioned (as in my suggestion) it makes more sense to position it using top, left etc. If not we would have to say "top-0 mt-1". 

This is how this looks for me:
<img width="192" alt="image" src="https://github.com/specklesystems/speckle-server/assets/139135120/246643d6-9b4c-47ad-a6a4-6dacb759def3">